packages/e2e-tests/src/specs/wordpress/blockWidget.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import {
+  deleteWidgets,
+  visitAdminPage,
+  activatePlugin,
+  deactivatePlugin,
+  checkVersion,
+  visitWidgetScreen,
+  insertWidget,
+} from '@web-stories-wp/e2e-test-utils';
+
+// TODO, Once 5.8 is released, update check for 'latest'.
+const maybe = checkVersion(process.env?.WP_VERSION, '5.8')
+  ? describe
+  : describe.skip;
+
+/* eslint-disable jest/require-top-level-describe, jest/consistent-test-it */
+maybe('Web Stories Widget Block', () => {
```

Comment:
In another PR I'm trying a different approach for this that I currently find a bit more readable.

In the `e2e-test-utils` package I created a new `conditions.js` file with some conditions like this:

```js
//eslint-disable-next-line jest/no-export
export function skipSuiteForWordPress58() {
  if (process.env.WP_VERSION.startsWith('5.8') {
    //eslint-disable-next-line jest/require-top-level-describe, jest/no-focused-tests
    test.only('does not work on this WordPress version', () => {});
  }
}
```

And then in the test I do:

```js
import { skipSuiteForWordPress58 } from '@web-stories-wp/e2e-test-utils';

describe('Web Stories Widget Block', () => {
  skipSuiteForWordPress58();

  // ...
});
```

Reply:
Works for me. Want me to make that change in this PR?

Reply:
Yeah 👍

Reply:
I decided to go a little different direction. My idea, combines both the ideas together in a nice workaround. 
`startsWith` simple will not work when WP 5.9 comes out. This will mean we have to change this on every major WP release. I have tweaked the check, to be a min WP version, I hope you like the change.